apps/settings/src/views/Users.vue

```diff
@@ -384,17 +386,21 @@ export default {
 		/**
 		 * Validate quota string to make sure it's a valid human file size
 		 *
-		 * @param {string} quota Quota in readable format '5 GB'
+		 * @param {string | object} quota Quota in readable format '5 GB' or Object {id: '5 GB', label: '5GB'}
 		 * @return {Promise|boolean}
```

Comment:
Doc type does not match anymore

```suggestion
		 * @return {object} The validated quota object or unlimited quota if input is invalid
```